packages/urlwatch.rb

```diff
@@ -0,0 +1,34 @@
+require 'package'
+
+class Urlwatch < Package
+  description 'A tool for monitoring webpages for updates'
+  homepage 'https://thp.io/2008/urlwatch/'
+  version '2.7'
+  source_url 'https://github.com/thp/urlwatch/archive/2.7.tar.gz'
+  source_sha256 '0a7b46d161501463dda99a1d82daf00237e2627616beaa6ccbf04d6c70c68ded'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'python3'
+
+  def self.preinstall
+    abort "Unable to install urlwatch.  /home/#{USER}/user/.local directory exists.".lightred if Dir.exist? "/home/#{USER}/user/.local"
```

Comment:
Composer and possibly other packages use subdirectories in this location. I don't think refusing to install if other packages have written data to ~/.local is the best way to do things.

Reply:
If you have a better suggestion on how this should be installed, I'm all ears.  What I don't want is for this package to end up being installed in `~/.local`. :)